presentation/src/main/java/com/example/healthysalad/SelectIngredientFragment.kt

```diff
@@ -8,67 +8,132 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.Button
+import android.widget.TextView
+import androidx.core.os.bundleOf
 import androidx.recyclerview.widget.GridLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import com.example.data.Data
-
+import kotlinx.android.synthetic.main.fragment_select_ingredient.view.*
 
 /**
  * A simple [Fragment] subclass.
  * Use the [SelectIngredientFragment.newInstance] factory method to
  * create an instance of this fragment.
  */
-class SelectIngredientFragment (ingredientType : String) : Fragment() {
-    private val ingredientType = ingredientType
+
+private const val INGREDIENTTYPE  = "base"
```

Comment:
I think for constants it's better for the variable names to include the value it's holding. For example, the name could be `INGREDIENTTYPE_BASE`. This would make it easier to understand what the value is when reading the code.

Reply:
it should still be ingredient type because it will change depending on the fragment. "base" is just the initialised value.